extension/README.md

```diff
@@ -1,2 +1,34 @@
-# vscode-xcomponent
-visual studio code extension for xcomponent
+# xcomponent
+
+This extension provides a development environment for [XComponent](http://www.xcomponent.com/).
+
+XComponent is a platform to easily create, monitor and share **microservices**. To get more details about XComponent, go to [the resource center](https://github.com/xcomponent/xcomponent).
+
+The extension includes the following features:
+* cxml preview
+
+More features yet to come...
+
+## Feature details
+
+## cxml preview
+
+You can display a graphical preview (state machine diagram) of a cxml file by using the *"XComponent: component preview"* command.
```

Comment:
To display a component *.cxml file use the *"XComponent: Preview Component"* command.